src/App.js

```diff
@@ -130,10 +133,9 @@ function App() {
     video.current.currentTime = startTime;
   }
 
-  // const handleTimer = () => {
-  //   setPlayState(!playState);
-  // };
-
+  function playTime() {
```

Comment:
play time is a value not a function we need to compute the correct timme and pass it as a number